nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/PersistenceContext.java

```diff
@@ -1486,7 +1486,7 @@ public Serializable copy(Node source, Serializable parentId, String name) {
         checkFreeName(parentId, name, complexProp(hierFragment));
         // do the copy
         Long pos = getNextPos(parentId, false);
-        CopyResult copyResult = mapper.copy(new IdWithTypes(source), parentId, name, null);
+        CopyResult copyResult = mapper.copy(new IdWithTypes(source), parentId, name, null, true);
```

Comment:
Could you do `boolean excludeSpecialChildren = true` then use `excludeSpecialChildren` in the `mapper.copy` call, to make things explicit. And add a comment on the local variable to explain why they are excluded (in this case because it's our policy, `no special children (comments) kept on document copy`)
Same for other calls to `mapper.copy` in this file, with suitable explanations.